.ops/build-docker/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:22.04
+
+ARG USER_ID=1001
+ARG GROUP_ID=1001
+
+ENV DEFAULT_JAVA_VER 11
+ENV JAVA_HOME /usr/lib/jvm/java-${DEFAULT_JAVA_VER}-openjdk-amd64/
+ENV DEBIAN_FRONTEND noninteractive
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+
+
+# tzdata ENVs required for installing software-properties-common
+ENV TZ=US/Eastern
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get -qq update && \
+    apt-get -y install \
+    curl \
+    docker-compose \
+    sudo \
+    ca-certificates \
+    openjdk-11-jdk \
+    vim \
+    build-essential \
+    maven \
+    cmake \
+    git \
+    openssh-client \
+    docker.io \
+    dnsutils \
+    wget \
+    telnet \
+    groff \
+    lcov \
+    bash-completion \
+    mingw-w64-x86-64-dev && \
```

Comment:
there might be some others in here you can still pare back to keep this small: docker-compose, docker.io in particular, but otherwise this is looking much better (and smaller)!

Reply:
Sure Paul. Will do that in next commit.